gitbase-playground/values.yaml

```diff
@@ -75,6 +75,7 @@ gitbaseServer:
   storage:
     size: 10Gi
   logLevel: debug
+  squashEnable: "false"
```

Comment:
From https://github.com/src-d/gitbase/blob/master/cmd/gitbase/main.go#L20 and https://github.com/src-d/gitbase/blob/master/cmd/gitbase/command/server.go#L39-L42 I understand that if you put something here, it will be enabled by default, even if what you put is the `"false"` string

Reply:
Thanks, I completely missed that from the readme. Fixed in 21c48eb, with the missing version bump.

Reply:
It wasn't at all clear... I had to go to the code to understand it